Makefile

```diff
@@ -86,3 +89,18 @@ docker-run:
 
 docker-run-db:
 	@docker-compose up db
+
+pre-build:
+	@rm -rf .build || true
+	@mkdir -p .build;
+	@rm -rf terraform/.artifacts || true
+	@mkdir -p terraform/.artifacts
+
+api-build: pre-build
+	@yarn install
+	@yarn workspace @symchk/api build
+	@yarn workspaces focus @symchk/api --production
+	@cp -r node_modules .build/node_modules
+	@cp -r ./apps/api/dist .build
```

Comment:
Maybe better to copy the contents of the dist folder than the dist folder itself?

Reply:
```suggestion
	@cp -r ./apps/api/dist/* .build
```